codec-http/src/main/java/io/netty/handler/codec/http/multipart/FileUploadUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.http.multipart;
+
+final class FileUploadUtil {
+
+    private FileUploadUtil() { }
+
+    static int hashCode(FileUpload upload) {
+        return upload.getName().hashCode();
+    }
+
+    static boolean equals(FileUpload upload1, FileUpload upload2) {
+        return upload1.getName().equalsIgnoreCase(upload2.getName());
+    }
+
+    static int compareTo(FileUpload upload1, FileUpload upload2) {
+        int v;
+        v = upload1.getName().compareToIgnoreCase(upload2.getName());
+        if (v != 0) {
+            return v;
+        }
+        // TODO should we compare size for instance ?
```

Comment:
do we have any context for how this was intended to be used? It seems strange to compare the length, but not compare the actual content. If we don't do this consider just making this method `return upload1.getName().compareToIgnoreCase(upload2.getName());`